evap/student/templates/student_vote.html

```diff
@@ -182,8 +191,8 @@ <h3 class="mb-3">{{ evaluation.full_name }} ({{ evaluation.course.semester.name
                     $("div.vote-inputs input[type=radio]:checked").parent().addClass("active");
 
                     // restore publish confirmation state
-                    if($("#text_results_publish_confirmation_top").prop('checked')) {
-                        $("#bottom_text_results_publish_confirmation_card").hide();
+                    if (textResultsPublishConfirmation.bottomCard) {
+                        updateTextResultsPublishConfirmation();
                     }
```

Comment:
Just a note why I extracted `updateTextResultsPublishConfirmation` here: Only asking for `textResultsPublishConfirmation.top.checked` will raise an error when the checkbox doesn’t exist and I thought that calling a shared function makes it clearer what state we are restoring